### PR TITLE
fix: omit showing Tooltip after Dialog or Drawer got closed

### PR DESCRIPTION
**Pull request by tujoworker**

### CODE REVIEW

File: packages/dnb-eufemia/src/components/modal/__tests__/Modal.test.tsx [hunk 1]
```diff
@@ -330,31 +330,55 @@ describe('Modal component', () => {
     ).toBe(true)
   })
 
+  it('will set "data-autofocus" attribute on focusing the trigger when closed', async () => {
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/modal/__tests__/Modal.test.tsx:333]: Anyone has a better text?

Comment (langz) [reply packages/dnb-eufemia/src/components/modal/__tests__/Modal.test.tsx:333]: Looks fine to me :)

File: packages/dnb-eufemia/src/components/modal/Modal.tsx [hunk 2]
```diff
@@ -285,20 +285,33 @@ class Modal extends React.PureComponent<
 
       this.setActiveState(this._id)
     } else if (modalActive === false) {
-      if (this._triggerRef && this._triggerRef.current) {
-        this._triggerRef.current.focus({ preventScroll: true })
+      const focus = (elem: HTMLElement) => {
+        // So we can omit showing a Tooltip when icon
```
Comment (langz) [inline packages/dnb-eufemia/src/components/modal/Modal.tsx:289]: _when icon_? Is that the correct text? 🤔

**Review by langz — APPROVED** [review]

**Pull request merged**